migration/incremental/015-tracing-redesign.sql

```diff
@@ -305,31 +305,25 @@ DECLARE
     _pg_version int4 := current_setting('server_version_num')::int4;
 BEGIN
     IF (_pg_version >= 140000) THEN
-        EXECUTE
-            'CREATE OR REPLACE FUNCTION ps_trace.tag_map_subscript_handler(internal) ' ||
-                'RETURNS internal '                 ||
-                'LANGUAGE internal '                ||
-                'IMMUTABLE PARALLEL SAFE STRICT '   ||
-                'AS $f$jsonb_subscript_handler$f$;'
-            ;
-
-        EXECUTE
-            'CREATE TYPE ps_trace.tag_map ( '       ||
-                'INPUT = ps_trace.tag_map_in, '     ||
-                'OUTPUT = ps_trace.tag_map_out, '   ||
-                'SEND = ps_trace.tag_map_send, '    ||
-                'RECEIVE = ps_trace.tag_map_recv, ' ||
-                'SUBSCRIPT = ps_trace.tag_map_subscript_handler);'
-            ;
+        CREATE OR REPLACE FUNCTION ps_trace.tag_map_subscript_handler(internal)
+                RETURNS internal
+                LANGUAGE internal
+                IMMUTABLE PARALLEL SAFE STRICT
+                AS $f$jsonb_subscript_handler$f$;
+
+        CREATE TYPE ps_trace.tag_map (
```

Comment:
Why is this type in `ps_trace`, but `tag_v` is in `_ps_trace`? Was that on purpose, or did it just happen?